src/test/resources/features/complaints/CCHBorderForce.feature

```diff
@@ -0,0 +1,14 @@
+@CCH @Complaints @BFComplaints
+Feature: CCH
+
+  @ComplaintsRegression2 @HOCS-4221
+  Scenario: User can transfer a BF complaints case to CCH
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "CCH" stage
```

Comment:
I think this step does truly detail what we are doing as it implies that the BF case will be at CCH stage, when the BF case is actually closed and a new COMP case is created. 
I would prefer something like an additional step:

When I create a "BF" case and move it to the "Triage" stage
And I do not accept the case
And I transfer the case to "CCH"